### PR TITLE
[zh] sync /news/support/announcing-1.16-eol/

### PR DESCRIPTION
**Pull request by jiecloud**
[zh] sync /news/support/announcing-1.16-eol/
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [ ] Docs
- [X] News
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/support/announcing-1.16-eol/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: 对 Istio 1.16 的支持将于 2023 年 7 月 19 日结束
+subtitle: 支持公告
+description: 即将发布的 Istio 1.16 生命周期结束公告。
+publishdate: 2023-06-19
+---
+
+根据 Istio 的 [支持政策]（/docs/releases/supported-releases#supported-releases/），
+像 1.16 这样的小版本在 N+2 小版本（在本例中为 1.18）发布后的六周内才会得到支持。
```
Comment (hanxiaop) [inline content/zh/news/support/announcing-1.16-eol/index.md:9]:
```suggestion
像 1.16 这样的小版本会在 N+2 小版本（在本例中为 1.18）发布后的六周内得到支持。
```

File: content/zh/news/support/announcing-1.16-eol/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: 对 Istio 1.16 的支持将于 2023 年 7 月 19 日结束
+subtitle: 支持公告
+description: 即将发布的 Istio 1.16 生命周期结束公告。
+publishdate: 2023-06-19
+---
+
+根据 Istio 的 [支持政策]（/docs/releases/supported-releases#supported-releases/），
```
Comment (windsonsea) [inline content/zh/news/support/announcing-1.16-eol/index.md:8]:
```suggestion
根据 Istio 的[支持政策](/zh/docs/releases/supported-releases#supported-releases/)，
```
不要把 url 的 () 改成汉语字符

File: content/zh/news/support/announcing-1.16-eol/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+像 1.16 这样的小版本在 N+2 小版本（在本例中为 1.18）发布后的六周内才会得到支持。
+[Istio 1.18 于 6 月 7 日发布]（/news/releases/1.18.x/announcing-1.18/），
```
Comment (windsonsea) [inline content/zh/news/support/announcing-1.16-eol/index.md:10]:
```suggestion
[Istio 1.18 于 6 月 7 日发布](/zh/news/releases/1.18.x/announcing-1.18/)，
```

**Review by windsonsea — COMMENTED** [review]
url 翻译后需增加 /zh，无论是否有对应中文

不要修改 url 的 () 括号字符为全角

File: content/zh/news/support/announcing-1.16-eol/index.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+对 1.16 的支持将于 2023 年 7 月 19 日结束。
+
+届时，我们将停止将安全问题和严重错误的修复程序向后移植到 1.16，
+因此我们鼓励您升级到最新版本的 Istio ({{<istio_release_name>}})。
```
Comment (my-git9) [inline content/zh/news/support/announcing-1.16-eol/index.md:14]: 行尾的空格需要删除

Comment (jiecloud) [reply content/zh/news/support/announcing-1.16-eol/index.md]: 已修改，谢谢提醒。

File: content/zh/news/support/announcing-1.16-eol/index.md (hunk 3 above)
Comment (jiecloud) [reply content/zh/news/support/announcing-1.16-eol/index.md:10]: 已修改，谢谢指正。

Comment (windsonsea) [reply content/zh/news/support/announcing-1.16-eol/index.md:10]: 额，没改呀，这 url 还是错的 -_!

File: content/zh/news/support/announcing-1.16-eol/index.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: 对 Istio 1.16 的支持将于 2023 年 7 月 19 日结束
+subtitle: 支持公告
+description: 即将发布的 Istio 1.16 生命周期结束公告。
+publishdate: 2023-06-19
+---
+
+根据 Istio 的 [支持政策](/zh/docs/releases/supported-releases#supported-releases/)，
+像 1.16 这样的小版本会在 N+2 小版本（在本例中为 1.18）发布后的六周内得到支持。
+[Istio 1.18 于 6 月 7 日发布]（/zh/news/releases/1.18.x/announcing-1.18/），
```
Comment (my-git9) [inline content/zh/news/support/announcing-1.16-eol/index.md:10]:
```suggestion
[Istio 1.18 于 6 月 7 日发布](/zh/news/releases/1.18.x/announcing-1.18/)，
```

**Review by my-git9 — CHANGES_REQUESTED** [review]

**Review by windsonsea — APPROVED** [review]
LGTM now.

If you have changed based on the suggested comments, you can click `Resolve conversation`, so each reviewer can easily know the PR progress.

**Pull request merged**